docker/install/ubuntu_install_vitis_ai_core.sh

```diff
@@ -22,8 +22,9 @@ set -o pipefail
 
 # install libraries for building Vitis-AI on ubuntu
 apt-get update && apt-get install -y --no-install-recommends \
-    graphviz\
-    gnupg2
+    graphviz \
+    gnupg2 \
+    gpg-agent
 
 apt-get update && apt-get install -y gcc-aarch64-linux-gnu
```

Comment:
I understand you removed it from the Dockerfile, to make it clear, and that was a good move. However, due to the way Docker layers works, you'll still have leftover files, which over time will bloat your image.

The code below is what I had in mind. _To be clear, it's a suggestion only._ Feel free to keep your script, if that makes more sense for you.

```suggestion
    graphviz \
    gnupg2 \
    gpg-agent \
    gcc-aarch64-linux-gnu \
    && rm -rf /var/lib/apt/lists/*
```

Reply:
Ok, thanks, I added `&& rm -rf /var/lib/apt/lists/*` and reorganized a bit. I didn't add your suggestion commit as I had to remove `--no-install-recommends` to avoid an issue with aarch64-linux-gnu-gcc returning the following when using it for cross compilation
```
/usr/lib/gcc-cross/aarch64-linux-gnu/7/../../../../aarch64-linux-gnu/bin/ld: cannot find crti.o: No such file or directory
/usr/lib/gcc-cross/aarch64-linux-gnu/7/../../../../aarch64-linux-gnu/bin/ld: cannot find -lc
/usr/lib/gcc-cross/aarch64-linux-gnu/7/../../../../aarch64-linux-gnu/bin/ld: cannot find crtn.o: No such file or directory
```
Ps  `rm -rf /var/lib/apt/lists/*` could be added to the other script files too, for example https://github.com/apache/tvm/blob/main/docker/install/ubuntu_install_core.sh

Reply:
Thanks for the update - Yeah, I agree there is some refactoring that can be done on other scripts as well.